### PR TITLE
Tag visibility testcases

### PR DESCRIPTION
**Pull request by rananda**
Tag visibility testcases

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 3c35c4b4e2e2155cb11db6461c32f25dd05a83a7:

`cfme/tests/cloud_infra_common/test_tag_visibility.py`:
- :bangbang: Line 36:101: **E501** *line too long (103 > 100 characters)*
- :red_circle: Line 38:21: **F821** *undefined name 'tag'*
- :red_circle: Line 38:47: **F821** *undefined name 'tag'*
- :red_circle: Line 51:5: **F821** *undefined name 'login'*
- :red_circle: Line 57:20: **F821** *undefined name 'tag'*
- :heavy_exclamation_mark: Line 61:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6c6f4141fc98c4b92666c5199091fc471f621507:

`cfme/tests/cloud_infra_common/test_tag_visibility.py`:
- :red_circle: Line 8:1: **F811** *redefinition of unused 'Credential' from line 4*
- :bangbang: Line 56:1: **E302** *expected 2 blank lines, found 1*
- :bangbang: Line 62:1: **E302** *expected 2 blank lines, found 1*
- :heavy_exclamation_mark: Line 66:1: **W391** *blank line at end of file*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8a5e574d0f2bd4094c8e2cb4e064bdd493a7dc43:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e7d718a616f6eaecd75702f9135a99fc4c5b9009:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/cloud_infra_common/test_tag_visibility.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    yield user
+    login.login_admin()
+    user.delete()
+
+
+@pytest.fixture(scope="module")
+def tagged_vm(new_tag, setup_infra_provider):
+    tag_vm = VM.factory('cu-9-5', setup_infra_provider)
+    tag_vm.add_tag(new_tag)
+    return tag_vm
```
Comment (psav) [inline cfme/tests/cloud_infra_common/test_tag_visibility.py]: This should be a ``yield``

File: cfme/tests/cloud_infra_common/test_tag_visibility.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+          display_name="tag_viz_" + fauxfactory.gen_alphanumeric().lower(),
+          category=category)
+    tag.create()
+    yield tag
+    tag.delete(cancel=False)
+
+
+@pytest.fixture(scope="module")
+def setup_infra_provider():
+    return get_crud('vsphere55')
```
Comment (psav) [inline cfme/tests/cloud_infra_common/test_tag_visibility.py]: We need to sort this out before we can merge

**Comment by dajoRH** [comment]
Lint report for commit fc65eacffa1af575ac0fe5e9ee337b288a46b9df:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit a9dc511cb2bac7e314c02d2c1ea26781aa6a27e5:

`cfme/tests/cloud_infra_common/test_tag_visibility.py`:
- :bangbang: Line 13:101: **E501** *line too long (102 > 100 characters)*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9510c7d01cea947b53009734b197f281936ee6da:

`cfme/tests/cloud_infra_common/test_tag_visibility.py`:
- :bangbang: Line 14:13: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit caeb16a31b43d2fd62e509af5cc8c094788475d3:

`cfme/tests/cloud_infra_common/test_tag_visibility.py`:
- :bangbang: Line 14:13: **E126** *continuation line over-indented for hanging indent*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit c92aa7e19174540de91c7fa73f8d08353eb5e82a:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 1c23bdb92712275f64b5df7f7ffcc98ca4832d22:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**